addons/binding/org.openhab.binding.minecraft/src/main/java/org/openhab/binding/minecraft/internal/discovery/MinecraftDiscoveryService.java

```diff
@@ -150,11 +162,8 @@ private void submitPlayerDiscoveryResults(ThingUID bridgeUID, String name) {
 
         Map<String, Object> properties = new HashMap<>();
         properties.put(MinecraftBindingConstants.PARAMETER_PLAYER_NAME, name);
-        thingDiscovered(DiscoveryResultBuilder.create(uid)
-                .withProperties(properties)
-                .withBridge(bridgeUID)
-                .withLabel(name)
-                .build());
+        thingDiscovered(DiscoveryResultBuilder.create(uid).withProperties(properties).withBridge(bridgeUID)
```

Comment:
pitty ;-)

Reply:
That part of the contribution was by the formatter and not me. :innocent: